modules/theme/src/themes/default/elements/label.overrides

```diff
@@ -131,25 +131,19 @@
 
 .ui.label {
     &.issuer-label {
-        color: #e68e00;
```

Comment:
Are these color changes expected?

Reply:
Please refer to https://github.com/wso2/identity-apps/pull/6993#issuecomment-2415817695